src/test/resources/mappings/twoDayTimeEntryDeleteAction.json

```diff
@@ -0,0 +1,139 @@
+          }
+        }
+      },
+      "response": {
+        "status": 200,
+        "jsonBody": {
+          "meta": {
+            "next": null
+          },
+          "items": [
+            {
+              "id": "c0a80193-87a3-1ff0-8187-a4277b0b0008",
+              "tenantId": "52a8188b-d41e-6768-19e9-09938016342f",
+              "personId": "b810a175-9b87-473c-94f3-9bcdf87f3a58",
+              "agreementId": "c0a80193-87a3-1ff0-8187-a3bfe2b80004",
+              "accrualDate": "2023-04-23",
+              "accrualTypeId": "e502eebb-4663-4e5b-9445-9a20441c18d9",
+              "cumulativeTotal": 8040,
+              "cumulativeTarget": 11520,
+              "contributions": {
+                "timeEntries": {
+                },
+                "total": 0.00
+              }
+            }
+          ]
+        },
+        "headers": {
+          "Content-Type": "application/json"
+        }
+      }
+    },
+    {
+      "request": {
+        "method": "GET",
+        "urlPathPattern": "/resources/accruals",
+        "queryParameters": {
+          "tenantId": {
+            "equalTo": "52a8188b-d41e-6768-19e9-09938016342f"
+          },
+          "filter": {
+            "equalTo": "personId=='b810a175-9b87-473c-94f3-9bcdf87f3a58'&&accrualDate<='2024-03-31'&&accrualDate>='2023-04-21'"
+          }
+        }
+      },
+      "response": {
+        "status": 200,
+        "jsonBody": {
+          "meta": {
+            "next": null
+          },
+          "items": [
+            {
+              "id": "c0a80193-87a3-1ff0-8187-a4277b0b0008",
+              "tenantId": "52a8188b-d41e-6768-19e9-09938016342f",
+              "personId": "b810a175-9b87-473c-94f3-9bcdf87f3a58",
+              "agreementId": "c0a80193-87a3-1ff0-8187-a3bfe2b80004",
+              "accrualDate": "2023-04-21",
+              "accrualTypeId": "e502eebb-4663-4e5b-9445-9a20441c18d9",
+              "cumulativeTotal": 8040,
+              "cumulativeTarget": 10560,
+              "contributions": {
+                "timeEntries": {
+                  "5afbfbeb-0d65-4e7e-bced-fb7e3a214c76": 480,
+                  "fee2523d-380e-4116-b2f3-d526d1b6ab55": 240
+                },
+                "total": 720
+              }
+            },
+            {
+              "id": "c0a80193-87a3-1ff0-8187-a4277b0b0008",
+              "tenantId": "52a8188b-d41e-6768-19e9-09938016342f",
+              "personId": "b810a175-9b87-473c-94f3-9bcdf87f3a58",
+              "agreementId": "c0a80193-87a3-1ff0-8187-a3bfe2b80004",
+              "accrualDate": "2023-04-22",
+              "accrualTypeId": "e502eebb-4663-4e5b-9445-9a20441c18d9",
+              "cumulativeTotal": 8460,
+              "cumulativeTarget": 11040,
+              "contributions": {
+                "timeEntries": {
+                  "a90a037a-a6c5-4fc0-9e72-a533c8c89c68" : 420
```

Comment:
Should this be 60 to represent the time entry being 23:00+01:00 to midnight? And also update the totals

Reply:
Updated to 60. The test data is now representative of the time entry being used in the test.